Traceback (most recent call last):
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.9/dist-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Run the function to find the antiderivative
antiderivative = find_antiderivative()

# Print the general antiderivative
print("General Antiderivative:")
print(antiderivative)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
Cell [0;32mIn[2], line 2[0m
[1;32m      1[0m [38;5;66;03m# Run the function to find the antiderivative[39;00m
[0;32m----> 2[0m antiderivative [38;5;241m=[39m [43mfind_antiderivative[49m[43m([49m[43m)[49m
[1;32m      4[0m [38;5;66;03m# Print the general antiderivative[39;00m
[1;32m      5[0m [38;5;28mprint[39m([38;5;124m"[39m[38;5;124mGeneral Antiderivative:[39m[38;5;124m"[39m)

Cell [0;32mIn[1], line 5[0m, in [0;36mfind_antiderivative[0;34m()[0m
[1;32m      3[0m [38;5;28;01mdef[39;00m [38;5;21mfind_antiderivative[39m():
[1;32m      4[0m     [38;5;66;03m# Prompt the user for the function f(x)[39;00m
[0;32m----> 5[0m     function_input [38;5;241m=[39m [38;5;28;43minput[39;49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43mEnter the function f(x): [39;49m[38;5;124;43m"[39;49m[43m)[49m
[1;32m      7[0m     [38;5;66;03m# Define the variable and function using sympy[39;00m
[1;32m      8[0m     x [38;5;241m=[39m sp[38;5;241m.[39mSymbol([38;5;124m'[39m[38;5;124mx[39m[38;5;124m'[39m)

File [0;32m/usr/local/lib/python3.9/dist-packages/ipykernel/kernelbase.py:1201[0m, in [0;36mKernel.raw_input[0;34m(self, prompt)[0m
[1;32m   1199[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39m_allow_stdin:
[1;32m   1200[0m     msg [38;5;241m=[39m [38;5;124m"[39m[38;5;124mraw_input was called, but this frontend does not support input requests.[39m[38;5;124m"[39m
[0;32m-> 1201[0m     [38;5;28;01mraise[39;00m StdinNotImplementedError(msg)
[1;32m   1202[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_input_request(
[1;32m   1203[0m     [38;5;28mstr[39m(prompt),
[1;32m   1204[0m     [38;5;28mself[39m[38;5;241m.[39m_parent_ident[[38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m],
[1;32m   1205[0m     [38;5;28mself[39m[38;5;241m.[39mget_parent([38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m),
[1;32m   1206[0m     password[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[1;32m   1207[0m )

[0;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.

